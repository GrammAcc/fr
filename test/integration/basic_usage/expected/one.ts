
// [[fr]]rename::something[[fr]]
export const something = "something"

export const somethingelse = "something"
