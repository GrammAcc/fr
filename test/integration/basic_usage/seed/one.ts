
// [[fr]]rename::something[[fr]]
export const something = "something"

// [[fr]]rename::someOtherThing::somethingelse[[fr]]
export const someOtherThing = "something"
