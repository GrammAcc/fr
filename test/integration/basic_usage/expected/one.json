{
  "something": 1,
  "somethingelse": "something"
}
